content/en/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -175,30 +172,131 @@ kubectl certificate approve my-svc.my-namespace
 certificatesigningrequest.certificates.k8s.io/my-svc.my-namespace approved
 ```
 
+You should now see the following:
 
-## Download the Certificate and Use It
+```shell
+kubectl get csr
+```
+
+```none
+NAME                  AGE   SIGNERNAME            REQUESTOR              REQUESTEDDURATION   CONDITION
+my-svc.my-namespace   10m   example.com/serving   yourname@example.com   <none>              Approved
+```
+
+This means the certificate request has been approved and is waiting for the
+requested signer to sign it.
+
+## Sign the Certificate Signing Request
+
+Next, we'll play the part of a certificate signer, issue the certificate, and upload it to the API.
```

Comment:
```suggestion
Next, you'll play the part of a certificate signer, issue the certificate, and upload it to the API.
```
([avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we))